tests/data/instances/sequence_tagging/verb_semantics_instance_test.py

```diff
@@ -0,0 +1,121 @@
+        data_indexer.add_word_to_index("O", namespace="tags")
+        data_indexer.add_word_to_index("B-ARG2", namespace="tags")
+
+        indexed_instance = instance.to_indexed_instance(data_indexer)
+        expected_sentence_indices = [roots_index, absorb_index, water_index]
+        expected_verb_indices = [0, 1, 0]
+        expected_entity_indices = [0, 0, 1]
+
+        assert_array_almost_equal(indexed_instance.sentence, expected_sentence_indices)
+        assert_array_almost_equal(indexed_instance.verb, expected_verb_indices)
+        assert_array_almost_equal(indexed_instance.entity, expected_entity_indices)
+
+        b_array = numpy.asarray([0, 1], dtype='int32')
+        o_array1 = numpy.asarray([1, 0], dtype='int32')
+        o_array2 = numpy.asarray([1, 0], dtype='int32')
+        expected_label = (numpy.asarray([1], dtype='int32'),
+                          numpy.asarray([b_array, o_array1, o_array2], dtype='int32'))
+        assert_array_almost_equal(indexed_instance.label[0], expected_label[0])
+        assert_array_almost_equal(indexed_instance.label[1], expected_label[1])
+
+        train_inputs, train_labels = indexed_instance.as_training_data()
+        expected_indices = (expected_sentence_indices, expected_verb_indices, expected_entity_indices)
+        assert_array_almost_equal(train_inputs[0], expected_indices[0])
+        assert_array_almost_equal(train_inputs[1], expected_indices[1])
+        assert_array_almost_equal(train_inputs[2], expected_indices[2])
+
+        assert_array_almost_equal(train_labels[0], expected_label[0])
+        assert_array_almost_equal(train_labels[1], expected_label[1])
+
+    def test_padding(self):
+        line_record = "roots####absorb####water\t1,1\t2,2\tMOVE\t-1,-1\t0,0"
+        data_indexer = DataIndexer()
+
+        instance = VerbSemanticsInstance.read_from_line(line_record)
+        roots_index = data_indexer.add_word_to_index("roots")
+        absorb_index = data_indexer.add_word_to_index("absorb")
+        water_index = data_indexer.add_word_to_index("water")
+        data_indexer.add_word_to_index("from")
+        data_indexer.add_word_to_index("soil")
+        padding_lengths = {'num_sentence_words': 5}
+        data_indexer.add_word_to_index("CREATE", namespace="state_changes")
+        data_indexer.add_word_to_index("DESTROY", namespace="state_changes")
+        data_indexer.add_word_to_index("MOVE", namespace="state_changes")
+        data_indexer.add_word_to_index("O", namespace="tags")
+        data_indexer.add_word_to_index("B-ARG2", namespace="tags")
+
+        indexed_instance = instance.to_indexed_instance(data_indexer)
+        indexed_instance.pad(padding_lengths)
+
+        expected_sentence_indices = [0, 0, roots_index, absorb_index, water_index]
+        expected_verb_indices = [0, 0, 0, 1, 0]
+        expected_entity_indices = [0, 0, 0, 0, 1]
+
+        b_array = numpy.asarray([0, 1], dtype='int32')
```

Comment:
In this case, it actually is the whole name, because "b" and "o" are tags.  It's like my `a_index` variable in other tests, for the word "a".